### PR TITLE
Replaces bleve with sqlite FTS5.

### PR DESCRIPTION
**Pull request by nopper**

### CODE REVIEW

File: internal/server/statvar/statvar_match.go [hunk 1]
```diff
@@ -16,39 +16,293 @@ package statvar
...
 
-func buildExplanationString(in *search.Explanation, sb *strings.Builder, level int) {
-	sb.WriteString(fmt.Sprintf("%.2f => %s\n", in.Value, in.Message))
-	if len(in.Children) > 0 {
-		for _, cchild := range in.Children {
-			for i := 0; i < level+1; i++ {
-				sb.WriteString(" ")
+const defaultLimit = 5
+
+const sqlCreateTables = `
```
Comment (shifucun) [inline internal/server/statvar/statvar_match.go:38]: Can you add some comments for these commands?

Comment (shifucun) [reply internal/server/statvar/statvar_match.go:38]: Maybe some higher level comments on how the sqlite search does?

**Review by shifucun — APPROVED** [review]
Can you add some comments for the rational for the change. Also what are the performance difference between the two? Is the index from sqlite smaller?

File: internal/server/statvar/statvar_match.go (hunk 1 above)
Comment (nopper) [reply internal/server/statvar/statvar_match.go:38]: Done.

**Pull request merged**